docs/DeveloperGuide.md

```diff
@@ -125,6 +125,41 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### \[Implemented\] Import timetable feature
+The user can import information about their lessons by giving their NUSMods timetable URL to ProductiveNUS to import.
+
```

Comment:
The phrasing sounds a little weird to me, though it could just be me XD The "to import" at the end of the sentence appears a bit redundant as well since at the start of the sentence, you already mentioned that the information will be imported.

Maybe "The user can import information about their lessons into ProductiveNUS using their NUSMods timetable URL" would be more concise?

Reply:
Amended :)

Reply:
Looks good :)